src/kits/tracker/ContainerWindow.cpp

```diff
@@ -1010,13 +968,41 @@ BContainerWindow::Init(const BMessage* message)
 	SetFlags(Flags() & ~B_NO_WORKSPACE_ACTIVATION);
 }
 
+void
+BContainerWindow::InitLayouts()
+{
+	BLayoutItem* item = 
+		fPoseViewHolder->GroupLayout()->AddView(0, fPoseView->TitleView());
+	BSize minSize = item->MinSize();
+	BSize maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(minSize.Width(), kTitleViewHeight));
+	item->SetExplicitMaxSize(BSize(maxSize.Width(), kTitleViewHeight));
+		
+	item = fCountContainer->GroupLayout()->AddView(fPoseView->CountView());
+	minSize = item->MinSize();
+	maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(kCountViewWidth, minSize.Height()));
+	item->SetExplicitMaxSize(BSize(kCountViewWidth, maxSize.Height()));
```

Comment:
No, without these it resizes them to far larger than they would be. These values were also used in the pre-layout code, I imagine removing them would require some significant rework of TitleView and CountView, which isn't really in the scope of this commit (it'd result in a behavioral change, too). But this commit will make it easier to do that in the future.